docs/sdd.md

```diff
@@ -239,7 +239,7 @@ sequenceDiagram
     scheduler->>scheduler: main control loop<br>create_itinerary_impl(&task)
     scheduler->>+storage: search(...)<br>1 Aircraft, 2 Vertipads Information
     storage->>scheduler: Records for the aircraft and<br>vertipads in proposed itinerary
-    scheduler->>scheduler: Confirm that itinerary is possible
+    scheduler->>scheduler: Confirm that itinerary is possible;<br>check intersections with existing zones<br>and flight paths
```

Comment:
This syntax is giving an error in the GitHub view
![Selection_425](https://github.com/user-attachments/assets/1d68693b-03fd-42c5-beba-59fe3b84133e)

Reply:
https://github.com/aetheric-oss/svc-scheduler/pull/53/commits/b9f523bb1322ac0661d75854c7e32764414a1b6a

Didn't like the semicolon